build-tools/automation/azure-pipelines.yaml

```diff
@@ -140,6 +141,20 @@ stages:
         artifactName: $(InstallerArtifactName)
         targetPath: bin/Build$(XA.Build.Configuration)/$(InstallerArtifactName)
 
+    - task: DotNetCoreCLI@2
+      displayName: pack Xamarin.Android.Sdk
+      inputs:
+        command: pack
+        packagesToPack: $(System.DefaultWorkingDirectory)/build-tools/create-packs/Packager.csproj
+        configuration: $(XA.Build.Configuration)
+        packDirectory: $(System.DefaultWorkingDirectory)/bin/packs
```

Comment:
Presumably `packDirectory` is the output directory.  Shouldn't this be e.g. `bin/Build$(XA.Build.Configuration)` as is done with the `.vsix` & `.pkg` files?

Reply:
I made this configuration agnostic so that we could easily use the same path in a test projects `NuGet.Config`: https://github.com/xamarin/xamarin-android/pull/3860/files#diff-6557e277ae88a841e6f9afa8423f9359R34

We could rework this at this time to use an azure devops feed instead however: https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/nuget?view=azure-devops&tabs=yaml#publish-your-packages

Reply:
A concern of mine with a "configuration agnostic" output directory is that if you locally build a Debug config + .nupkg, then build a Release config + .nupkg, the second build will overwrite the first.  (I assume; I can't easily determine  what the`.nupkg` filename would be based on `Packager.csproj`.

As such, I *think* I'd prefer to leave the Configuration within the output path, unless it would otherwise be within the `.nupkg` filename (though I'm not sure that would actually be better in practice).

Reply:
Fair point, I'll look at publishing to an internal feed instead. That way we don't need to explicitly download the .nupkg for our test jobs.

Reply:
After further consideration/investigation, I think we should hold off on feed distribution for the moment as it is dependent on unique versioning. Considering that the next phase of investigation will be breaking up this NuGet package, I don't think it makes sense to set up a robust versioning schema at this time. I've updated output and pack directories to instead contain `$(Configuration)` information for now.